### PR TITLE
Mz/bench integer compression

### PR DESCRIPTION
**Pull request by mayeul-zama**

### CODE REVIEW

File: tfhe/benches/integer/glwe_packing_compression.rs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    let mut bench_group = c.benchmark_group(bench_name);
+
+    let cks = ClientKey::new(param);
+
+    let private_compression_key = cks.new_compression_private_key(comp_param);
+
+    let (compression_key, decompression_key) =
+        cks.new_compression_decompression_keys(&private_compression_key);
+
+    for num_blocks in [4, 8, 16, 32, 64, 128] {
```
[the remark this answers is not in the document]
Comment (mayeul-zama) [reply tfhe/benches/integer/glwe_packing_compression.rs]: Done

File: tfhe/benches/integer/glwe_packing_compression.rs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    let mut bench_group = c.benchmark_group(bench_name);
+
+    let cks = ClientKey::new(param);
+
+    let private_compression_key = cks.new_compression_private_key(comp_param);
+
+    let (compression_key, decompression_key) =
+        cks.new_compression_decompression_keys(&private_compression_key);
+
+    for num_bits in [8, 16, 32, 64, 128, 256] {
```
Comment (IceTDrinker) [inline tfhe/benches/integer/glwe_packing_compression.rs:23]: doest his also cover the max capacity of the GLWE ?

Comment (mayeul-zama) [reply tfhe/benches/integer/glwe_packing_compression.rs:23]: Yes, `lwe_per_glwe` is 256

Comment (IceTDrinker) [reply tfhe/benches/integer/glwe_packing_compression.rs:23]: except this is a number of bits so we would have 128 bodies, we should have a separate case that builds a number of bits for the max capcity 

like

256, lwe_per_glwe * message_modulus.0.ilog2()

Comment (mayeul-zama) [reply tfhe/benches/integer/glwe_packing_compression.rs:23]: Indeed

**Review by IceTDrinker — APPROVED** [review]
Thanks a lot !

**Pull request merged**